.cloud-build/cleanup/resource_cleanup_manager.py

```diff
@@ -156,3 +157,29 @@ class BatchPredictionJobCleanupManager(VertexAIResourceCleanupManager):
 
 class ExperimentCleanupManager(VertexAIResourceCleanupManager):
     vertex_ai_resource = aiplatform.Experiment
+
+class BucketCleanupManager(VertexAIResourceCleanupManager):
+    vertex_ai_resource = storage.bucket.Bucket
+
+    def list(self) -> Any:
+        storage_client = storage.Client()
+        return [ bucket for bucket in storage_client.list_buckets()]
+
+    def delete(self, resouce):
+        if not 'tests' in resouce.name:
```

Comment:
Decision to delete doesn't go in the `delete` method. See the `is_deletable` method in the base class. In any case, the established rule is already defined there (i.e. resource.name.startswith("temp")), so you can just remove this.